src/cmr/concepts/grid.js

```diff
@@ -0,0 +1,38 @@
+import Concept from './concept'
+
+export default class Grid extends Concept {
+  /**
+   * Instantiates a grid object
+   * @param {Object} headers HTTP headers provided by the query
+   * @param {Object} requestInfo Parsed data pertaining to the Graph query
+   * @param {Object} params GraphQL query parameters
+   */
+  constructor(headers, requestInfo, params) {
+    super('grids', headers, requestInfo, params)
+  }
+
+  /**
+   * @param {Object} jsonResponse HTTP response from the CMR endpoint
```

Comment:
This comment needs an explanation, can just use the same one all the other `parseJsonBody` methods use 
```   
* Parse and return the array of data from the nested response body
```